py/server/deephaven/utils.py

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
I am very strongly against `utils` modules.  They are dumping grounds that never end up well.  This should be a `gc` module, a `system` module, in the `deephaven` module, or something else that is specific.